jsapp/js/stores/session.ts

```diff
@@ -89,6 +82,20 @@ class SessionStore {
     );
   }
 
+  public logOut() {
+    dataInterface.logout().then(
+      action('logOutSuccess', () => {
+        // Reload so a new CSRF token is issued
+        window.setTimeout(() => {
+          window.location.replace('');
+        }, 1);
+      }),
+      action('logOutFailed', () => {
+        console.error('logout failed for some reason. what should happen now?');
+      }),
+    );
+  }
```

Comment:
Dupes [actions.es6's actions.auth.logOut](https://github.com/kobotoolbox/kpi/blob/5d1238085e4415ae2cc1888d887d0372512decf5/jsapp/js/actions.es6#L388-L399C3) used by the [Accountbox menu link](https://github.com/kobotoolbox/kpi/blob/5d1238085e4415ae2cc1888d887d0372512decf5/jsapp/js/components/header/accountMenu.tsx#L150). 

Should these be consolidated?